### PR TITLE
Add .gitignore, improve README, handle PRs with review requests of hidden teams

### PR DESCRIPTION
**Pull request by bm1549**

### CODE REVIEW

File: lib/github.rb [hunk 1]
```diff
@@ -55,7 +55,9 @@ def self._pr_data(pr)
     end
 
     result["reviewRequests"] = pr["reviewRequests"]["nodes"].map do |rr|
-      { "user" => name_and_login(rr["requestedReviewer"]) }
+      if !rr["requestedReviewer"].nil?
```
Comment (jasonpenny) [inline lib/github.rb:58]: Do you have an example PR where this broke? There should be no nodes returned if the `requestedReviewer` is nil

Comment (bm1549) [reply lib/github.rb:58]: The failing PR is this one: https://github.com/Appboy/currents-users-events/pull/216

Comment (bm1549) [reply lib/github.rb:58]: It seems that one of my team members has contributions outside of Braze being picked up

Comment (bm1549) [reply lib/github.rb:58]: It seems that one of my team members has contributions outside of my organization being picked up

Comment (jasonpenny) [reply lib/github.rb:58]: I see, there is a review request, but I don't have access to that team and apparently the API is returning null even though the team name is visible in the web UI

**Review by jasonpenny — APPROVED** [review]

**Pull request merged**